debugger/src/main/java/org/codice/acdebugger/breakpoints/SecurityCheckInformation.java

```diff
@@ -397,7 +394,12 @@ private int findNextToFindNotAlreadyFound(int lastNextToFind, List<String> conte
     int nextToFind = lastNextToFind;
 
     while (++nextToFind < size) {
-      if (contextDomains.indexOf(contextDomains.get(nextToFind)) > lastNextToFind) {
+      final String nextDomainToFind = contextDomains.get(nextToFind);
+
+      if (nextDomainToFind == null) {
```

Comment:
ℹ️ This account for boot domain classes in the middle of the stack which should be skipped right away.